libobs/util/c99defs.h

```diff
@@ -50,7 +50,8 @@
 #ifdef _MSC_VER
 #define EXPORT __declspec(dllexport)
 #else
-#define EXPORT
+/*  This is a GNU C-ism; might not work in clang.  */
+#define EXPORT __attribute__((visibility("default")))
```

Comment:
Clang seems to have it, so I think the comment can be removed right ?
Multiple version docs about this:
- https://releases.llvm.org/8.0.0/tools/clang/docs/LTOVisibility.html
- https://releases.llvm.org/10.0.0/tools/clang/docs/LTOVisibility.html
- https://clang.llvm.org/docs/LTOVisibility.html